README.md

```diff
@@ -1,2 +1,28 @@
-# github robot
-github robot
+# ci-bot
+[![Go Report Card](https://goreportcard.com/badge/github.com/Huawei-PaaS/ci-bot)](https://goreportcard.com/badge/github.com/Huawei-PaaS/ci-bot)
+[![Build Status](https://travis-ci.org/Huawei-PaaS/ci-bot.svg?branch=master)](https://travis-ci.org/Huawei-PaaS/ci-bot)
+[![LICENSE](https://img.shields.io/badge/license-Apache%202-blue.svg)](https://github.com/Huawei-PaaS/ci-bot/blob/master/LICENSE)
+
+This repository houses ci bot which aims to provide more simple and rich features for the github project users.
```

Comment:
What about outline the usage of ci bot?

This repository houses CI robot which will run your tests, add your labels, and merge your codes.

Reply:
Thanks, I will update it.